Makefile

```diff
@@ -1,14 +1,12 @@
 PLUGINS := example-cli example-getter example-postrender example-legacy-cli example-legacy-downloader
 
 helm4:
-	$(eval TMP := $(shell mktemp -d))
-	@pushd $(TMP) \
-		&& git clone git@github.com:scottrigby/helm.git \
-		&& cd helm \
-		&& git checkout plugin-types \
-		&& $(MAKE) build \
-		&& popd \
-		&& mv $(TMP)/helm/bin/helm helm4
+    # A suitable version of Helm source needs to be checked out and built at ../helm
```

Comment:
i did the clone thing in a temp dir out of sheer laziness since we were moving quickly. this looks fine tho :+1: